test/Constraints/moveonly_constraints.swift

```diff
@@ -42,7 +42,7 @@ enum Maybe<T> {
   case just(T)
 }
 
-func takeConcrete(_ m: __shared MO) {}
+func takeConcrete(_ m: borrowing MO) {}
```

Comment:
Should we keep tests of `__shared` as well?   I expected these to be duplicated, not replaced.

Reply:
The only difference in between shared and borrowing is the mangling. Otherwise, I would have duplicated the tests like I did with consuming.